src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasRelationSlicer.java

```diff
@@ -576,6 +669,28 @@ private String getShardOrAtlasName()
         return countryEntityMap;
     }
 
+    /**
+     * Check if the {@link Geometry} should be filtered out based on the provided bound.
+     *
+     * @param geometry
+     *            The {@link Geometry} to check.
+     * @return {@code true} if the given geometry should be filtered out.
+     */
+    private boolean isOutsideWorkingBound(final Geometry geometry)
```

Comment:
This looks identical to the method in `RawAtlasPointAndLineSlicer` - consider pulling it out from both into the `RawAtlasSlicer`.

Reply:
good idea!